### ISSUE TITLE
Command line interface? (CLI)

### ISSUE THREAD

**Issue by zeke** [transitive-bullshit/chatgpt-api#362]
### Describe the feature

It would be kinda cool if I could `npx chatgpt "prompt"` and have it print the result to standard output. Then I could do stuff like this:

```
npx chatgpt "write a shell command that takes input.mp4, speeds it up 5x, and outputs fast.mp4" | bash
```

I think it would just be a matter of adding a script like this:

```js
import { ChatGPTAPI } from './wherever/it/is'

const input = process.argv[2]

if (!input) {
  console.log('Usage: chatgpt "input prompt"')
  process.exit()
}

const api = new ChatGPTAPI({ apiKey: process.env.OPENAI_API_KEY })
const res = await api.sendMessage(input)
process.stdout.write(res.text)
```

Then setting that script as `bin` in `package.json`

What do you think, @transitive-bullshit?

**Comment by transitive-bullshit** [comment]
This is a great idea @zeke 💪 

As long as the core package stays relatively lightweight, I'm all for adding CLI functionality. `npx chatgpt "prompt"` is an excellent motivating use case.

Will take a look at #363 shortly.

**Comment by transitive-bullshit** [comment]
I got excited about this and ran with it 😄 

Upgrade to [v4.7.0](https://github.com/transitive-bullshit/chatgpt-api/releases/tag/v4.7.0) to try it out (see the new CLI section in the readme for usage).

@zeke I ran into some shell oddities with `process.stdout.write` when it's optimized for piping, but I want to make sure the defaults work when running from the shell without piping. E.g., if I just use `process.stdout.write` without adding any newline at the end, then Terminal on mac at least gets cutoff and doesn't show the stdout. I've also noticed this happening with some other CLIs like `curl`, so it could be an issue with my shell settings, though I'd bet other folks will run into this as well.

Anyhow, thanks for the great idea & PR 🙏 

I'd view this as an initial version, and I'm very open to suggestions / PRs for improving things.
